platform/feature-api/api/src/pod-api/tests/uri.test.ts

```diff
@@ -0,0 +1,20 @@
+import { createUUID, isPolypodUri, PolyUri } from "../uri";
+
+describe("Test functions", () => {
+    it("generates and UUID in the required format", () => {
+        expect(createUUID()).toMatch(/^\w{8}-\w{4}-4\w{3}-\w{4}-\w{12}$/);
+    });
+    it("generates different UUIDs", () => {
+        let lastUUID = createUUID();
+        for (let i = 0; i < 100; i++) {
+            const thisUUID = createUUID();
+            expect(thisUUID).not.toEqual(lastUUID);
+            lastUUID = thisUUID;
+        }
+    });
+    it("recognizes good and bad URIs", () => {
+        const polyUri: PolyUri = new PolyUri();
+        expect(isPolypodUri(polyUri.toString())).toBe(true);
+        expect(isPolypodUri("foobargaz")).toBe(false);
+    });
```

Comment:
nit: (I know these tests are quite simple)
but this smells like being two different test cases
where we want to populate some valid polyUris and verify that `isPolypodUri` returns `true`
and some invalid ones where it returns `false`...

Reply:
It would retest the same code path over and over...

Reply:
e.g. or 
```suggestion
  it.each([
    [new PolyUri(), true],
    ["foobargaz", false],
    ["badURI", false],
    ["xxxx-xxxx-xxx-xxxxxx", false],
    ["", false],
  ])(
    `should return proper result when passed arguments are: %i, %i`,
    (x, result) => {
      expect(isPolypodUri(x)).toEqual(result);
    }
  );

```

Reply:
> It would retest the same code path over and over...

well, there's no difference on what you're doing now in fact, as it doesn't need a prior setup or mock. 
we're on tests anyway, this is normal and expected. every `it` to be atomic 
(but, well, I also prefer the `it.each()` way if you fancy it)